course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -63,14 +74,19 @@ if name == best_friend:
 
 ### The `else` keyword
 - the `else` keyword is used to run a block of code if all the other `if`/`elif` statements have failed before it
-- when using the `else` keyword there shouldn't be any condition following it, only the block of code after it
-- the syntax for using the `else` keyword is simple:
-    1. write the `else` keyword in the same indent as the last `if`/`elif` statement
-    2. follow the keyword with colon
-    3. indented block of code that will get executed if other statements fail
+- the `else` keyword is followed by a colon without any condition before it
+- syntax for using the `else` keyword:
```

Comment:
Here since you've got the title, we can shorten the bullet points a bit, and probably delete the third bullet point.

### The `else` keyword
- Used to run a block of code if all the other `if`/`elif` statements have failed before it
- We don't specify a condition with this keyword

Reply:
Changed this, resolving